auth/server/src/main/java/com/seagates3/controller/AccountController.java

```diff
@@ -44,12 +42,10 @@
 import com.seagates3.s3service.S3AccountNotifier;
```

Comment:
If no changes in file then remove the file from commit.

Reply:
done.. other changes are there which are of removing unused packages warnings and code-formatter fixes